src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--28">
+      <div class="calendar__day-1 calendar__day-1--start-day--sun"></div>
```

Comment:
1. do not give exactly which day of month it is (`1`, `2`), you need to do it in scss (use loop)
2. modifier `start-day--sun` should be on parent element, not here